GDCpp/GDCpp/Extensions/Builtin/RuntimeSceneTools.cpp

```diff
@@ -71,6 +71,10 @@ bool GD_API SceneJustBegins(RuntimeScene &scene) {
   return scene.GetTimeManager().IsFirstLoop();
 }
 
+bool GD_API SceneJustResumed(RuntimeScene &scene) {
+  return scene.SceneJustResumed();

```

Comment:
You started to implement this for the C++ game engine, but the C++'s RuntimeScene does not have this method. I would suggest to not implement this for GDCpp. Instead:
remove this for RuntimeSceneTools.cpp/.h and remove the declaration from `GDCpp/GDCpp/Extensions/Builtin/SceneExtension.cpp` too.
When `SetFunctionName` is not called for a "platform" (GDCpp here), the IDE won't show the function if the game is using this "platform".